node/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "extends": "../shared/tsconfig",
   "compilerOptions": {
     "paths": {
-      "jslib-common/*": ["../common/src/*"]
+      "jslib-common/*": ["../common/src/*"],
+      "jslib-node/*": ["./src/*"]
```

Comment:
If we move to tests alongside tested code, no. It's nice here and now because it means I don't have 200 files with path changes from `paths` to relative.

I also think the `paths` are better than the current form of relative path, which would be traversing all the way up to the root of the project, then back down through `src` to wherever the file is.

Reply:
Right, it's fine to leave as is, but I think we should have a discussion around where to put tests but that can be handled in a separate PR.

Reply:
I do too, I like the idea of alongside the main file and excluded from prod by a `*.spec.*` matcher. It would solve both the path replacement and the weird relatives problems nicely